### ISSUE TITLE
Lots of warnings in idea.log about issue parsing error

### ISSUE THREAD

**Issue by VladRassokhin** [jk1/youtrack-idea-plugin#84]
idea.log is just spammed with messages and logs rotates too oftenly:
```text
2018-04-25 14:57:52,218 [7239354]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,218 [7239354]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,218 [7239354]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,218 [7239354]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,219 [7239355]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
2018-04-25 14:57:52,220 [7239356]   WARN -        com.github.jk1.ytplugin - YouTrack issue parse error. Offending element: {"name":"wikified","value":"true"} 
```

IDE: IntelliJ IDEA (build #IU-181.4445.78, 09 Apr 2018 23:12)
YouTrack Integration Plugin (0.9.1)

**Comment by jk1** [comment]
@VladRassokhin is it reproducible for you with 0.9.2?

**Comment by jk1** [comment]
With no further feedback provided, I consider it fixed in 0.9.2
